spec/integration/outputs/elasticsearch/node_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "spec/es_spec_helper"
```

Comment:
you should call this using ```require_relative``` otherwise when running the test from logstash it will fail.